.env.example

```diff
@@ -91,19 +94,19 @@ BEE_CODE_INTERPRETER_CA_CERT= # optional
 BEE_CODE_INTERPRETER_CERT= # optional
 BEE_CODE_INTERPRETER_KEY= # optional
 # Storage for code interpreter, possible values are: s3, filesystem
-BEE_CODE_INTERPRETER_STORAGE_BACKEND=s3
+BEE_CODE_INTERPRETER_STORAGE_BACKEND=filesystem
 # applicable for s3 storage, uses S3 credential above
 BEE_CODE_INTERPRETER_BUCKET_FILE_STORAGE=
 # applicable for filesystem storage
-BEE_CODE_INTERPRETER_FILE_STORAGE_PATH=
+BEE_CODE_INTERPRETER_FILE_STORAGE_PATH: ../bee-stack/tmp/code-interpreter-storage
```

Comment:
wouldn't it be better to use the actual `/tmp` directory?

Reply:
I'm not sure when it gets cleared, you'd get this error in bee-stack when the folder you're trying to a folder into docker container that does not exist:

```
Error response from daemon: error while creating mount source path '/tmp/code-interpreter-storage': chown /tmp/code-interpreter-storage: permission denied
```

so if tmp gets cleared on reboot and docker containers persist through reboots.
We have `restart: always` on this container but not on others, perhaps this is something we should look at.
Anyway I think this is safer :D